### ISSUE TITLE
The release changelog GHA CI removed and restored the `1.*` changelogs

### ISSUE THREAD

**Issue by Icarus9913** [kumahq/kuma#10783]
### What happened?

The PR [#10770](https://github.com/kumahq/kuma/pull/10770) removed the `1.*` changelogs and later #10771  restored it. 
It's pretty strange.

**Comment by Icarus9913** [comment]
https://github.com/Kong/kong-mesh/pull/6278/files
The Kong-mesh met this too, but after I triggered `Release 0.0.0` it turned to correct.

**Comment by lahabana** [comment]
Are we missing a release of ci-tools which includes: https://github.com/kumahq/ci-tools/pull/49 ?

**Comment by slonka** [comment]
https://github.com/kumahq/kuma/commit/6e283139b2ec9cc6c6e849dd971a3fde85fedcfb - I think we bumped it with @bartsmykla

**Comment by lahabana** [comment]
Indeed! it's curious... Might be worth looking at the 2 action logs to see what happened...

**Comment by Icarus9913** [comment]
IIRC, we removed the `1.*` versions for out-of-date manually. So we can close this issue right now
